src/include/miopen/ctc.hpp

```diff
@@ -56,7 +56,7 @@ struct CTCLossDescriptor : miopenCTCLossDescriptor
                                    const int* inputLengths,
                                    miopenCTCLossAlgo_t algo) const;
 
-    void CTCLoss(const Handle& handle,
+    void CTCLoss(Handle& handle,
```

Comment:
Yes it does. CTC uses softmax and passes handle for softmax calculation. Since this handle is no longer a const for softmax we need to remove const modifier here as well. If we have other options lets discuss.

Reply:
@Vsevolod1983 is it so that we need to make many other changes in the code in order to keep the handle `const`?